### PR TITLE
Add NPN to an SSL3 build

### PR DESCRIPTION
**Pull request by richsalz**

### CODE REVIEW

**Comment by richsalz** [comment]
ping @ekasper :)

File: .travis.yml [hunk 1]
```diff
@@ -45,7 +45,7 @@ matrix:
           env: CONFIG_OPTS="enable-msan"
         - os: linux
           compiler: clang-3.6
-          env: CONFIG_OPTS="no-asm enable-ubsan enable-rc5 enable-md2 enable-ssl3 enable-ssl3-method -fno-sanitize=alignment"
+          env: CONFIG_OPTS="no-asm enable-ubsan enable-rc5 enable-md2 enable-ssl3 enable-ssl3-method enable-nextprotoneg -fno-sanitize=alignment"
```
Comment (ekasper) [inline .travis.yml]: Can you please also enable it on the COVERALLS target above?

Comment (richsalz) [inline .travis.yml]: somehow i missed your comment.  added your request.

**Comment by richsalz** [comment]
thanks!  commit 211bf9f

**Pull request closed without merging**